dwave/system/composites/reversecomposite.py

```diff
@@ -0,0 +1,205 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# =============================================================================
+"""
+Composites that do batch operations for reverse annealing.
+"""
+
+import dimod
+import numpy as np
+
+__all__ = 'ReverseAdvanceComposite', 'ReverseBatchStatesComposite'
+
+
+class ReverseAdvanceComposite(dimod.ComposedSampler):
+    """Composite that reverse anneals an initial sample through a sequence of anneal
+     schedules. If you don not specify an initial sample, a random sample is used for the first
```

Comment:
"don not".

Also, can we break this into a single sentence at the top and then a longer paragraph describing the behaviour? The sphinx docs ideally like a short summary.